ReactAndroid/src/main/java/com/facebook/react/shell/MainReactPackage.java

```diff
@@ -95,6 +96,12 @@ public NativeModule get() {
           return new AppStateModule(context);
         }
       }),
+      new ModuleSpec(BlobModule.class, new Provider<NativeModule>() {
+        @Override
+        public NativeModule get() {
+          return new BlobModule(context);
+        }
+      }),
```

Comment:
OK makes sense to include it in all open source apps by default - this adds a bit to app size even if people don't use it but we do that for all "standard" modules already.